lib/puppet-strings/yard/code_objects/type.rb

```diff
@@ -78,7 +78,7 @@ class Feature
     # @param [String] docstring The docstring of the feature.
     def initialize(name, docstring)
       @name = name
-      @docstring = docstring
+      @docstring = Puppet::Util::Docs.scrub(docstring).gsub("\n", ' ')
```

Comment:
Note: `Puppet::Util::Doc.scrub` doesn't strip newlines, so it seems we have to do it ourselves.